Content.Server/Traits/TraitSystem.cs

```diff
@@ -15,25 +17,52 @@ public override void Initialize()
         base.Initialize();
 
         SubscribeLocalEvent<PlayerSpawnCompleteEvent>(OnPlayerSpawnComplete);
+        SubscribeLocalEvent<BeenClonedEvent>(OnBeenCloned);
     }
 
+    /// <summary>
+    ///     All the jobs that can't have traits.
+    /// </summary>
     private readonly List<string> _jobBlacklist = new()
     {
         "SAI",
         "Cyborg",
         "MedicalCyborg"
     };
```

Comment:
Feel like this should either be a YAML, or a true/false in the YAML for the role or entity.